kolibri/core/tasks/utils.py

```diff
@@ -276,6 +277,11 @@ def update_progress(self, increment=1, message="", extra_data=None):
             )
             if extra_data and isinstance(extra_data, dict):
                 self.job.update_metadata(**extra_data)
+        if self.download_request:
+            self.download_request.update_progress(
```

Comment:
This is a global mixin, not just used for downloading individual pieces of metadata - this seems like the wrong place to make this change.

It would be better instead to subclass the `RemoteChannelResourceImportManager` class specifically for a content request import, and add this logic there, invoking the `super` method for `update_progress` and then also applying this change.